apps/client/src/app/pages/home/components/footer/Footer.tsx

```diff
@@ -0,0 +1,31 @@
+import Button from '@map-shared-components/Button';
+import SocialMediaButtons from './components/SocialMediaButtons';
+
+function Footer({ oneClick }: FooterProps) {
+  return (
+    <footer className='bg-[rgb(29,60,102)] text-[#CADEED] py-1 px-3  mx-auto w-full]'>
+      <div className='container mx-auto flex justify-center  className=w-full sm:w-1/2 md:w-auto m-0.5'>
+        <div className='text-center font-roboto '>
+          <SocialMediaButtons />
+          <div>
+            <Button onClick={oneClick}>Info</Button>
```

Comment:
This Button repeats 5 times in this component create a common component for these, please